src/components/BookingForm/PersonParticularsStep/BookingParticulars.jsx

```diff
@@ -9,25 +9,6 @@ import React, { useEffect, useState } from "react";
 import { useValidation } from "../../../contexts/ValidationContext";
 import { useSession } from "next-auth/react";
 
-export const FormTwoColumn = styled(Box)(({ theme }) => ({
```

Comment:
move this to .style file, if you don't know about this text me on slack we will have aa short call